editor/src/core/shared/github.ts

```diff
@@ -742,3 +743,12 @@ export async function refreshGithubData(
     dispatch([updateGithubData(emptyGithubData())], 'everyone')
   }
 }
+
+export function disconnectGithubProjectActions(): EditorAction[] {
+  return [
+    updateGithubData(emptyGithubData()),
+    updateGithubChecksums({}),
+    updateBranchContents(null),
+    updateGithubSettings({ ...emptyGithubSettings() }),
```

Comment:
```suggestion
    updateGithubSettings(emptyGithubSettings()),
```
Looks like an unnecessary spread here

Reply:
https://github.com/concrete-utopia/utopia/pull/2794/commits/7d8939f442ee0485d2f70622d98ea7bcdd68fd68